### DISCUSSION TITLE
Mixing ./build_openroad.sh and ./build_openroad.sh --local

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#754] [General] [answered]
I ran ```./build_openroad.sh --local``` on Ubuntu 22 and then ```./build_openroad.sh``` to generate a docker image.

The docker image contained a yosys-abc that was using libreadline.so.8, same as Ubuntu.

Is there a safe way to use the same folder to create docker images and do local builds, or should I really have two separate folders?

**Comment by vvbandeira** [comment, marked as answer]
I think we work under the assumption that a user of ORFS would use only one method, docker or local build, depending on their preference and environment restrictions. Could you share the reason why you need both docker and local installs?

**Reply by oharboe** [reply]
I see. Noted.

When I debug and develop pull requests, I use -local.

When I create docker images for internal consumption with our build system, I use docker.

**Reply by vvbandeira** [reply]
Understood. For now, I would suggest using different folders for local and docker to avoid "contamination", but I will look into supporting your use case. Thanks for the feedback.

**Reply by oharboe** [reply]
This is just a slight hickup that is easily worked around.

This issue is much more significant as it messes up the development machine for non OpenROAD projects:


https://github.com/The-OpenROAD-Project/OpenROAD/issues/2699
